bs-react-native-next/src/apis/NetInfo.md

```diff
@@ -0,0 +1,48 @@
+---
+id: apis/NetInfo
+title: NetInfo
+wip: true
+---
+
+```reason
+type info = {
+  .
+  "type_": ConnectionType.t,
+  "effectiveType": EffectiveConnectionType.t,
+};
+
+type remove = {
```

Comment:
Why did you define your own `remove` when we already have `EventSubscription.t` in a common `EventSubscription` module?

Reply:
I checked the implementation in RN code and some addEventListener functions have output type `EmitterSubscription` whereas NetInfo and BackHandler define `{remove: ....}`.

Reply:
Ah, ok, interesting. Then let's keep it as it is!